### PR TITLE
[WLA iOS] Add logo for Dolby Atmos to asset details view

### PR DESCRIPTION
**Pull request by delarge77**
[WLA iOS] Add logo for Dolby Atmos to asset details view

### CODE REVIEW

File: Sources/iOSClientExposure/Asset/Response/AssetFeature.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        
+        id = try container.decodeIfPresent(String.self, forKey: .id)
+        images = try container.decodeIfPresent([AssetFeatureImage].self, forKey: .images)
+    }
+    
+    internal enum CodingKeys: String, CodingKey {
+        case id
+        case images
+    }
+}
```
Comment (udayasri) [inline Sources/iOSClientExposure/Asset/Response/AssetFeature.swift:34]: Where do you use this `AssetFeature` ? Don't see any API response that actually use this struct

File: README.md [hunk 2]
```diff
@@ -21,6 +22,7 @@
 * [Upgrade Guides](#upgrade-guides)
 
 ## Features
+- [x] Asset Features
```
Comment (udayasri) [inline README.md:25]: Again , asset feature is not really a feature , remove it here as well. Otherwise it can misguide the developers

**Review by udayasri — APPROVED** [review]
Fix the comments, otherwise LGTM

**Review by shobaki — APPROVED** [review]
👍

**Pull request merged**